<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BasalContactsWidget</class>
 <widget class="QWidget" name="BasalContactsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Basal Contacts Extractor</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="geologyLayerLabel">
       <property name="text">
        <string>Geology Layer:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QgsMapLayerComboBox" name="geologyLayerComboBox">
       <property name="filters">
        <set>QgsMapLayerProxyModel::PolygonLayer</set>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="unitNameFieldLabel">
       <property name="text">
        <string>Unit Name Field:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QgsFieldComboBox" name="unitNameFieldComboBox"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="formationFieldLabel">
       <property name="text">
        <string>Formation Field:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QgsFieldComboBox" name="formationFieldComboBox"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="faultsLayerLabel">
       <property name="text">
        <string>Faults Layer:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QgsMapLayerComboBox" name="faultsLayerComboBox">
       <property name="filters">
        <set>QgsMapLayerProxyModel::LineLayer</set>
       </property>
       <property name="allowEmptyLayer">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="stratiColumnLabel">
       <property name="text">
        <string>Stratigraphic Order Layer:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QgsMapLayerComboBox" name="stratiColumnComboBox"/>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="ignoreUnitsLabel">
       <property name="text">
        <string>Units to Ignore:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="ignoreUnitsLineEdit">
       <property name="placeholderText">
        <string>Comma-separated list of units to ignore</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="runButton">
     <property name="text">
      <string>Extract Basal Contacts</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsMapLayerComboBox</class>
   <extends>QComboBox</extends>
   <header>qgsmaplayercombobox.h</header>
  </customwidget>
  <customwidget>
   <class>QgsFieldComboBox</class>
   <extends>QComboBox</extends>
   <header>qgsfieldcombobox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
